pages/orders/[id].vue

```diff
@@ -104,7 +104,7 @@ const activities = [
                 </el-timeline-item>
               </el-timeline>
             </div>
-            <div class="w-full h-60 pl-4 flex flex-col items-center justify-center">
+            <div class="w-full h-60 pl-4 flex flex-col items-center justify-center z-1">
```

Comment:
Are we sure this appears above the button too?

Reply:
The buttons to zoom in and out are still visible if that's what you mean